UnityProject/Assets/Scripts/Items/Implants/Organs/Liver.cs

```diff
@@ -14,7 +14,7 @@ public class Liver : BodyPartModification
 		/// <summary>
 		///ReagentContainer which the liver uses to hold reagents it will process. Reagents like alcohol will be broken down into their ethanol reagent via ReactionSet
 		/// </summary>
-		private ReagentContainer processingContainer;
+		[HideInInspector] public ReagentContainer processingContainer;
```

Comment:
I would say using `NonSerialized` is more appropriate here, because we don't need anything to persist on disk.